ansible_ai_connect/ai/api/permissions.py

```diff
@@ -132,6 +132,7 @@ class BlockUserWithoutSeat(permissions.BasePermission):
     message = "User doesn't have access to the IBM watsonx Code Assistant."
 
     def has_permission(self, request, view):
+        return CONTINUE
```

Comment:
Is this correct?

You're circumventing the `Permission`.

Was it just for testing?

Reply:
Yes, you are right it was added to make end-to-end tests easier for the reviewer. Removing it now. Thank you!

P.S. Let's use `ANSIBLE_AI_ENABLE_TECH_PREVIEW`=True instead.